src/sys/socket/mod.rs

```diff
@@ -241,6 +247,12 @@ impl SockProtocol {
     #[cfg(apple_targets)]
     #[allow(non_upper_case_globals)]
     pub const KextEvent: SockProtocol = SockProtocol::Icmp; // Matches libc::SYSPROTO_EVENT
+
+    /// Packet filter on IPv4 traffic
+    /// NOTE: placed here due to conflict (little endian arch) with SockProtocol::NetLinkISCI
```

Comment:
@SteveLauC let me know which direction you prefer me to take here. e.g. using this for all these Eth* flags to avoid future conflicts, not using it all ...

Reply:
The current approach looks good, i.e., when add them via associated constants when we see conflicts